### ISSUE TITLE
Getting nan after few iteration

### ISSUE THREAD

**Issue by PRAYALANKAR** [yxlijun/DSFD.pytorch#17]
Hi 
I am trying to run the code but after few iterations i am getting the nan in the loss. Apart from the data paths i have kept everything same.
Any suggestions ?

**Comment by mafamily2496** [comment]
It's easy to get nan, start another train after getting nan

**Comment by Haoxl1997** [comment]
I have the same problem

**Comment by Wujielun** [comment]
> I have the same problem

Hi HaoSir,  I also meet the problem, do you have any solution now?
